packages/dnb-eufemia/src/extensions/forms/Field/__tests__/PhoneNumber.test.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { wait } from '../../../../core/jest/jestSetup'
+import { fireEvent, render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import PhoneNumber from '../PhoneNumber'
+
+describe('Field.PhoneNumber', () => {
+  it('should return correct value onFocus and onBlur event', async () => {
+    const onFocus = jest.fn()
+    const onBlur = jest.fn()
+    render(<PhoneNumber onFocus={onFocus} onBlur={onBlur} />)
+
+    const phoneElement = document.querySelector(
+      '.dnb-forms-field-phone-number__number input'
+    )
+
+    fireEvent.focus(phoneElement)
+    expect(onFocus).toHaveBeenLastCalledWith(undefined)
+
+    fireEvent.blur(phoneElement)
+    expect(onBlur).toHaveBeenLastCalledWith(undefined)
+
+    await userEvent.type(phoneElement, '99999999')
+
+    fireEvent.focus(phoneElement)
+    expect(onFocus).toHaveBeenLastCalledWith('99999999')
+
+    fireEvent.blur(phoneElement)
+    expect(onBlur).toHaveBeenLastCalledWith('99999999')
+  })
+
+  it('should return correct value onChange event', async () => {
+    const onChange = jest.fn()
+    const onCountryCodeChange = jest.fn()
+    render(
+      <PhoneNumber
+        onChange={onChange}
+        onCountryCodeChange={onCountryCodeChange}
+      />
+    )
+
+    const phoneElement = document.querySelector(
+      '.dnb-forms-field-phone-number__number input'
+    )
+    await userEvent.type(phoneElement, '99999999')
+    expect(onChange).toHaveBeenLastCalledWith('99999999')
+
```

Comment:
If you empty the field, then the component emits `undefined` and not an empty string. Maybe it is intentional, but I did expect it to be an empty string.

```suggestion

    await userEvent.clear(phoneElement)
    expect(onChange).toHaveBeenLastCalledWith('')

    await userEvent.type(phoneElement, '99999999')
```

Reply:
I agree, but would like to hear @henit what he thinks.